hardware/raspberrypi/bootmodes/msd.md

```diff
@@ -4,7 +4,19 @@
 
 This tutorial explains how to boot your Raspberry Pi from a USB mass storage device such as a flash drive or a USB hard disk. Note that this feature does not work with all USB mass storage devices.
 
-## Program USB host boot mode (Raspberry Pi 3A+ and 3B only)
+See the [documentation](./) for the boot sequence and alternative boot modes (Network, USB device, GPIO or SD boot).
+
+Note that "USB Mass Storage Boot" is different from "USB Device Boot Mode", the device boot mode allows a Raspberry Pi connected to a computer to boot using files on that computer.
```

Comment:
I wonder if the comma here should instead be a semi-colon?

Reply:
I've changed it to a `.` 
I'm not sure what fits best as I'm not a native English speaker, but I feel there is no use in concatenating both sentences.